lucene-linguistics/src/test/java/com/yahoo/language/lucene/LuceneTokenizerTest.java

```diff
@@ -10,10 +10,7 @@
 import org.junit.Test;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
We want to avoid glob imports because they may lead to breaking compatibility when the imported package adds something to its namespace.

Reply:
Intellij 🤦 just a second